avaframe/com1DFA/com1DFA.py

```diff
@@ -93,6 +93,71 @@ def copyReplace(origFile, workFile, searchString, replString):
         file.write(fileData)
 
 
+def getSimulation(cfg, rel, entResInfo):
+    """ get Simulation to run for a given release
```

Comment:
Bit unclear what is meant here. Maybe 
Given a simulation scenario, returns the types of simulations that need to be performed

Reply:
I changed this to: Get a list of all simulations that shall be performed according to simTypeList in configuration file;
        and a dictionary with information on release area